plugins/devtools/README.md

```diff
@@ -147,7 +147,7 @@ To use the permission framework to secure the DevTools sidebar option you'll wan
 
    ```sh
    # From your Backstage root directory
-   yarn add --cwd packages/app @backstage/plugin-devtools
+   yarn add --cwd packages/app @backstage/plugin-devtools-common
```

Comment:
one line above on line 146 it says to add the plugin-devtools-common you need to add it....

so this just fixes the correct package name